Automattic/wp-calypso
# Site profiler: Remove status icon from the heading block

Closes https://github.com/Automattic/wp-calypso/issues/82356

## Proposed Changes

* Remove status icon from the heading block

## Testing Instructions



* Got to `/site-profiler`
* Enter any valid domain
* Check if there is only domain without badge in the main heading block

## Pre-merge Checklist
